src/test/java/cucumber/steps/TasksSteps.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2021 JalaSoft.
+ * This software is the confidential and proprietary information of JalaSoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with JalaSoft
+ *
+ * @author Gustavo Huanca
+ */
+
+package cucumber.steps;
+
+import clickup.ApiEndpoints;
+import clickup.entities.features.tasks.Task;
+import clickup.entities.features.tasks.Tasks;
+import clickup.utils.ScenarioContext;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import core.api.ApiManager;
+import core.api.ApiMethod;
+import core.api.ApiRequest;
+import core.api.ApiRequestBuilder;
+import core.api.ApiResponse;
+import core.api.ApiHeaders;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import org.testng.asserts.SoftAssert;
+import java.util.LinkedList;
+
+import static core.utils.RandomCustom.random;
+
+public class TasksSteps {
+    private ApiRequestBuilder apiRequestBuilder = new ApiRequestBuilder();    private ApiRequest apiRequest;
+    private ApiResponse apiResponse;
+    private SoftAssert softAssert = new SoftAssert();
+    private ScenarioContext scenarioContext = ScenarioContext.getInstance();
+    private ApiResponse response = new ApiResponse();
+
+    public TasksSteps(ApiResponse apiResponse) {
+        this.apiResponse = apiResponse;
+    }
+
+    @Given("I add the amount of {int} to the total of task")
+    public void createNFeatures(int amount)
```

Comment:
Is possible to use the TaskRequest class instead of  build a new apirequest?

Reply:
Yes, but I don't use that because I create a list and delete that not to waste time to delete every task whish I create